extensions/interactions/LogicProof/directives/logic-proof-help-modal.directive.html

```diff
@@ -10,7 +10,7 @@
   <p><i>Just use the lines you have seen so far.</i></p>
   <p>
     <div class="row">
```

Comment:
Can you try wrapping this div inside a container div? 
divs with the row class need to be wrapped inside a container to work properly. I think that would be the right fix. Please refer: https://getbootstrap.com/docs/4.1/layout/grid/

Reply:
Thanks for the suggestion! Done!